data_gpu/driver/README.md

```diff
@@ -22,7 +22,7 @@ $ sudo systemctl disable sddm    # For SDDM
 $ sudo systemctl disable nvidia-persistenced
 ```
 
-Install nvidia cuda packages and cuda toolkit:
+Install the nvidia 555 drivers and the cuda 12.3 toolkit:
 
 ```bash
 $ wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/cuda-keyring_1.0-1_all.deb
```

Comment:
@JJL772 Are we sure `sudo dpkg -i cuda-keyring_1.0-1_all.deb` still needed? Or just the 'fog of war` step?

Reply:
This is still needed because the 555 drivers and cuda 12.3 come from the NVIDIA repository. This will help us keep CUDA and driver versions consistent across machines with different distros.